kolibri/core/templatetags/core_tags.py

```diff
@@ -71,3 +72,12 @@ def theme_favicon():
     favicon_url = favicon_urls[0] if favicon_urls else static("assets/logo.ico")
 
     return format_html('<link rel="shortcut icon" href="{}">', favicon_url)
+
+
+@register.simple_tag()
+def site_title():
+    """
+    Return the text of the site title, if provided by the theme. If not, the
+    default will be returned.
+    """
+    return ThemeHook.get_theme().get("siteTitle", _("Kolibri"))
```

Comment:
My translation knowledge is shaky at best, but does this support translation of the title in the theme?

Reply:
As far as I understand things, yes, the theme should support translation. The theme is [just a Django plugin](https://github.com/endlessm/kolibri-endless-key-theme/blob/main/kolibri_endless_key_theme/kolibri_plugin.py) with a hook which returns a dict of values. It should be able to call `_()` on the value it returns for `siteTitle`.

(That would mean we need to set up gettext support for kolibri-endless-key-theme, but that’s fairly straightforward.)